Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml

```diff
@@ -34,6 +34,8 @@
       - id: CableMVStack
       - id: trayScanner
         prob: 0.9
+      - id: NetworkConfigurator
```

Comment:
the toolbox has a capacity of 60 that gets breached by 5 in the rare ocassion everything spawns

Reply:
Should I or group it with the t-ray scanner?

Reply:
idk, this toolbox is more for messing with power so a tray makes way more sense, i would just not put it in there